public/index.html

```diff
@@ -655,16 +672,43 @@ <h5 class="widgetSubTitle" style="margin-bottom: 20px">
 					'</br>' +
 					document.getElementById(id).outerHTML.replace(/</g, '&lt;') +
 					'</code>';
+				if (widgetIframe) {
+					let circlebgcolor = "";
+          			let primarycolor = "";
+          				if (document.getElementById(id).circlebgcolor) {
+            				circlebgcolor = document.getElementById(id).circlebgcolor.replace("#", "");
+         				 }
+          				if (document.getElementById(id).primarycolor) {
+           					primarycolor = document.getElementById(id).primarycolor.replace("#", "");
+    }
+					widgetIframe.innerHTML =
+					'<code>' +
+					encodeURI(window.location.origin + '/iframe' +
+					'?type=' + document.getElementById(id).nodeName.toLowerCase() +
+					'&user=' + document.getElementById(id).user +
+					'&tenantkey=' + document.getElementById(id).tenantkey +
+					'&theme=' + document.getElementById(id).theme +
+					'&primarycolor=' + primarycolor +
+					'&circlebgcolor=' + circlebgcolor +
+					'&community=' + document.getElementById(id).community +
+					'&locale=' + document.getElementById(id).locale +
+					//'&refresh=' + document.getElementById(id).refresh +
+					'&forestname=' + (document.getElementById(id).forestname || '') +
+					'&project=' + (document.getElementById(id).project || '') // +
+					//'&goal=' + (document.getElementById(id).goal || '')
+					).replace(/#/g, '%23') +
+					'</code>';
```

Comment:
Looks mostly fine. It would be good to avoid keeping empty query parameters in the url though. Can that be done?